src/backend/distributed/sql/udfs/citus_finish_pg_upgrade/latest.sql

```diff
@@ -203,9 +203,6 @@ BEGIN
         'n' as deptype
     FROM pg_catalog.pg_dist_partition p;
 
-    -- set dependencies for columnar table access method
-    PERFORM columnar_internal.columnar_ensure_am_depends_catalog();
-
```

Comment:
Note to reviewer:

We should not assume presence of citus_columnar anymore.

Actually, these lines should've never been placed in a Citus UDF because even before it was possible that a user could drop citus_columnar later on, so should've never made such an assumption.

Reply:
Now, a new UDF is created for columnar at src/backend/columnar/sql/udfs/columnar_finish_pg_upgrade/13.2-1.sql and invocation of "columnar_internal.columnar_ensure_am_depends_catalog()" is moved there.

Also note that now "columnar_finish_pg_upgrade()" needs to be called by users after a PG upgrade, as it was the case for citus_finish_pg_upgrade().